### ISSUE TITLE
Jenkins failing with `build/common.sh: No such file or directory`

### ISSUE THREAD

**Issue by nikhiljindal** [kubernetes/kubernetes#19044]
Logs:
```
go run ./hack/e2e.go -v --down
18:37:00 2015/12/22 18:37:00 e2e.go:303: Running: teardown
18:37:00 /jenkins-master-data/jobs/kubernetes-pull-build-test-e2e-gce/workspace@2/kubernetes/hack/e2e-internal/../../cluster/gce/../../cluster/common.sh: line 29: /jenkins-master-data/jobs/kubernetes-pull-build-test-e2e-gce/workspace@2/kubernetes/hack/e2e-internal/../../cluster/gce/../../cluster/../build/common.sh: No such file or directory
18:37:00 2015/12/22 18:37:00 e2e.go:309: Error running teardown: exit status 1
```

http://kubekins.dls.corp.google.com:8081/job/kubernetes-pull-build-test-e2e-gce/21725
http://kubekins.dls.corp.google.com:8081/job/kubernetes-pull-build-test-e2e-gce/21724

cc @kubernetes/goog-testing

**Comment by nikhiljindal** [comment]
https://github.com/kubernetes/kubernetes/pull/19045 is out to fix this

**Comment by wojtek-t** [comment]
I've just merged it manually - hopefully will fix the problem.

@k8s-oncall - FYI

**Comment by fgrzadkowski** [comment]
This was fixed by reverting #18893 in #19045
